README.md

```diff
@@ -148,73 +148,29 @@ _Note that options were changed in version 0.5.3_
 For example these commands:
 
 ```python
-worldengine world -s 4 -n an_example -p 2048 -q 25 -x 2048 -y 2048
+worldengine -b -s 4 -n an_example -q 25 -x 2048 -y 2048
 ```
 
 Produce this output
 
 ```
-Worldengine - world generator
+Worldengine - a world generator (v. 0.18.0)
 -----------------------
- seed              : 4
- name              : seed3
- width             : 2048
- height            : 2048
- plates resolution : 2048
- number of plates  : 25
- operation         : world generation
- step              : full
+ operation              : world generation
```

Comment:
perhaps we could display the output as it was correctly aligned, I will send a PR to correctly align it